libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -1819,7 +1819,11 @@ GCS_MAVLINK::update_receive(uint32_t max_time_us)
 
     // send a timesync message every 10 seconds; this is for data
     // collection purposes
+    #if HAL_HIGH_LATENCY2_ENABLED
```

Comment:
nitpick: normally we have no indenting before #if.  E.g. see "#if AP_SCRIPTING_ENABLED" higher up in the file..

Reply:
FIxed